extension-manifest-v3/src/store/options.js

```diff
@@ -45,9 +45,9 @@ export const ENGINES = [
 
 const Options = {
   // Main features
-  blockAds: false,
```

Comment:
couldn't this be achieved fully on model level? eg. `blockAds` getter could take `terms` into consideration

Reply:
It could, but the main features are already implemented. Even if there will be something, that needs that condition, I think it is worth it.